### PR TITLE
Update app feeds

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-641

Relates to https://github.com/eReolen/base/pull/64 which moves the App feed module to “base”.

### CODE REVIEW

File: ereol_inspiration/ereol_inspiration.features.field_instance.inc [hunk 1]
```diff
@@ -117,15 +231,17 @@ function ereol_inspiration_field_default_field_instances() {
...
   );
 
   // Translatables
   // Included for use with string extractors like potx.
+  t('App feed color');
+  t('App feed image');
   t('Category query');
   t('Elementer');
-  t('Query for the main category');
+  t('Søgestreng for hovedkategorien');
```
Comment (cableman) [inline ereol_inspiration/ereol_inspiration.features.field_instance.inc]: All strings should be in english in the code

**Review by cableman — CHANGES_REQUESTED** [review]
This also changes fields on an content type and  adds a new image style?

File: ereol_inspiration/ereol_inspiration.features.inc [hunk 2]
```diff
@@ -13,6 +13,34 @@ function ereol_inspiration_ctools_plugin_api($module = NULL, $api = NULL) {
   }
 }
 
+/**
+ * Implements hook_image_default_styles().
+ */
+function ereol_inspiration_image_default_styles() {
+  $styles = array();
+
+  // Exported image style: app_feed_image.
```
Comment (cableman) [inline ereol_inspiration/ereol_inspiration.features.inc]: This image style is also found in https://github.com/eReolen/ereolengo/pull/7/files#diff-5546ba06abf6b3ce90a2fc5794b8ddfdR19

It should have been defined in the reol_app_feeds module as shared code and removed from the site modules.

Comment (rimi-itk) [reply ereol_inspiration/ereol_inspiration.features.inc]: Fixed.

File: ereol_inspiration/ereol_inspiration.features.field_instance.inc (hunk 1 above)
Comment (rimi-itk) [reply ereol_inspiration/ereol_inspiration.features.field_instance.inc]: Fixed.

**Review by cableman — APPROVED** [review]

**Pull request merged**